src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private final Operation operation;
+    private final String fruit;
+    private final int quantity;
+
+    public FruitTransaction(Operation operation, String fruit, int quantity) {
+        this.operation = operation;
+        this.fruit = fruit;
+        this.quantity = quantity;
+    }
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public enum Operation {
```

Comment:
The `Operation` enum is an inner class of `FruitTransaction` and should be declared in a separate file to follow the single responsibility principle. This will make the code more organized and maintainable.